app/src/main/java/co/nimblehq/recentapps/thumbnailhiding/MainActivity.kt

```diff
@@ -4,10 +4,11 @@ import android.app.Activity
 import android.os.Bundle
 import android.view.View.GONE
 import android.view.View.VISIBLE
-import androidx.appcompat.app.AppCompatActivity
 import kotlinx.android.synthetic.main.activity_main.*
 
-class MainActivity : AppCompatActivity(), RecentAppsThumbnailHidingListener {
+class MainActivity : RecentAppsThumbnailHidingActivity() {
+
+    override val enableSecureFlagOnLowApiDevices: Boolean = true
```

Comment:
Optional: `isSecureFlagOnLowApiDevicesEnabled`

Reply:
@Tuubz this is a var to be override so it looks like a setter, I think keep it as `enableSecureFlagOnLowApiDevices` will make more sense, `isSecureFlagOnLowApiDevicesEnabled` sounds a getter